CAMS-CSM1-0-tasmax.yaml ssps fail to validate biascorrected validation, missing timesteps"

HIstorical passes but all SSPs fail.

Archived workflow is here: https://argo.cildc6.org/archived-workflows/default/cae83af3-457a-45b1-95cb-c9aaec4454bf

One of the logged errors:
```
Validating gs://biascorrected-492e989a/stage/ScenarioMIP/CAMS/CAMS-CSM1-0/ssp370/r2i1p1f1/day/tasmax/gn/v20211202061048.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 658, in _test_timesteps
    assert (
AssertionError: projection bias_corrected file is missing timesteps, has 31025
```

This is blocking progress on #201.

--- a/workflows/templates/qualitycontrol-check-cmip6.yaml
+++ b/workflows/templates/qualitycontrol-check-cmip6.yaml
@@ -40,7 +40,7 @@ spec:
           - name: data
           - name: time
       script:
-        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.11.0
+        image: us-central1-docker.pkg.dev/downscalecmip6/private/dodola:0.11.1
         command: [python]
         source: |
           import dask

